controllers/apply/awards-for-all/form.js

```diff
@@ -87,15 +87,20 @@ module.exports = function({
                     <p><a href="/funding/programmes/awards-for-all-northern-ireland">You'll need to download an application form to apply</a>.</p>
                     <p>Only applicants in Scotland can apply through our new online form at the moment. We're working on making this available for the rest of the UK.</p>
                 `,
-                cy: ``
+                cy: `
+                    <p><a href="https://apply.tnlcommunityfund.org.uk">Byddwch angen defnyddio’r ffurflen hon yn lle</a>.</p>
+                    <p><strong>Ymgeisio am brosiect yng Ngogledd Iwerddon?</strong></p>
+                    <p><a href="/welsh/funding/programmes/awards-for-all-northern-ireland">Byddwch angen lawrlwytho ffurflen gais i ymgeisio</a>.</p>
```

Comment:
Highlighting the hyperlink change to make sure it exists.

Reply:
That one doesn't (we don't need to translate country-specific content that isn't Wales-related – so this link should just be `/funding/programmes/awards-for-all-northern-ireland`

Reply:
Got it! Have changed this, will push it with the next commit